lib/browser/lighthouse.js

```diff
@@ -174,3 +174,42 @@ exports.assertLighthouseScore = function assertLighthouseScore(
 exports.runLighthouseAudit = function runLighthouseAudit(flags, config) {
   return this.getLighthouseData(flags, config || lhConfig).then(parseLhResult);
 };
+
+exports.a11yAudit = function a11yAudit(flags, config) {
+  return this.getLighthouseData(flags, config || lhConfig).then(result => {
+    const violations = result.artifacts.Accessibility.violations;
+    return violations.reduce((a11yResult, violation) => {
+      const {
+        description,
+        nodes,
+        snippet,
+        impact,
+        help,
+        helpUrl,
+        id,
+        tags,
+      } = violation;
+
+      return a11yResult.concat(
+        nodes.map(node =>
+          Object.assign(
+            {
+              auditId: id,
+              description,
+              snippet,
+              impact,
+              help,
+              helpUrl,
+              tags,
+            },
+            {
+              selector: Array.isArray(node.target) ? node.target.join(' ') : '',
+              path: node.path,
+              snippet: node.snippet,
+            }
+          )
+        )
+      );
+    }, []);
```

Comment:
FFR, you can also fake a flatMap with something like:
```js
  return [].concat(
    ...violations.map(violation => {
      const { nodes /*, ... */ } = violation;
      return nodes.map(node => Object.assign(/* ... */));
    })
  );
```

Reply:
Nice, I thought we couldn't use `...`